### DISCUSSION TITLE
How do I get a 260MeV/u 2cmSOBP carbon ion beam with matRad

### DISCUSSION THREAD

**Discussion by Yuri-HubL** [e0404/matRad#635] [Q&A] [answered]
Hi, everybody.

I wonder if anyone knows how to obtain RBE weighted SOBP (260MeV/u 2cm SOBP) using matRad.

your sincerely,
Yuri

**Comment by wahln** [comment, marked as answer]
SOBPs are in general easy to create with a box-like phantom. For example, just taking the matRad.m script, and change it to load the BOXPHANTOM, loading carbon ions instead of photons, choose only one gantry angle and RBExD for optimization with LEMIV model (you can also check the examples, there is one for carbon ion planning that shows the respective setting), and so on.

A little bit more difficult is to get the exact 260MeV/u SOBP. Tips:
- check out the first example. It has code to create a box-like phantom from scratch with any size / parameters you want (will need a little bit of understanding the code and tweaking before it is 2cm at the right depth).
- The right depth can be obtained by getting the range for 260MeV carbon ions. You can load the carbon_Generic.mat file and look in machine.data to find the corresponding range

**Comment by Yuri-HubL** [comment]
Hi wahl，

Thank you very much for your answer. It was very helpful. I looked at the data from carbon_Generic.mat, which seems to be all monoenergetic carbon ion beams, but I want to get broadened carbon ion beams, I don't know if there is an algorithm in matRad that can get different weights of monoenergetic beams.

Getting the right weights seems to be the crux of the problem.

your sincerely
Yuri

**Comment by Yuri-HubL** [comment]
Hi, everybody.

I have successfully achieved my goal SOBP (range 105 weight 2cm). I will attach my file below in hopes of helping you.
1. As wahl says, the correct range can be found in the carbon_Generic.mat file
2. Construct a phantom and place it above the position associated with the range, making sure it is the width you want
[SOBP.txt](https://github.com/e0404/matRad/files/11765588/SOBP.txt)

your sincerely,
Yuri
